docs_src/performance.md

```diff
@@ -11,13 +11,13 @@ make benchmark
 ```
 
 ```bash
-make PIPELINE_SCRIPT=yolov5s.sh PIPELINE_COUNT=2 benchmark 
+make PIPELINE_COUNT=2 benchmark 
```

Comment:
do we need a section to explain how to specify a different pipeline other than the default?
or do we have that section already?